### ISSUE TITLE
debug failing pdx imports

### ISSUE THREAD

**Issue by sheridancbio** [knowledgesystems/pipelines-scrum#1123]
## Done Condition (What do we need? Why do we need it? Keep this is small as possible!)
pdx imports are succeeding nightly again

**Comment by sheridancbio** [comment]
logs show that the call to subset_and_merge_crdb_pdx_studies.py was failing:
usage: subset_and_merge_crdb_pdx_studies.py [-h] -c
                                            CLINICAL_ANNOTATION_MAPPING_FILE
                                            -d DATA_SOURCE_DIRECTORIES -f
                                            FETCH_DIRECTORY -l LIB -m
                                            MAPPING_FILE -r ROOT_DIRECTORY -t
                                            TEMP_DIRECTORY -w WARNING_FILE -a
                                            ANNOTATOR -s SAMPLE_LISTS_CONFIG
subset_and_merge_crdb_pdx_studies.py: error: argument -a/--annotator: expected one argument

@averyniceday identified the problem : the annotator.jar file is required for pdx runs and is missing. This is because the active/production copy of annotator.jar was coming out of the target directory in the clone of genome-nexus-annotation-pipeline. Last week I attempted to rebuild portals, which failed. Trying to debug I attempted to build (with maven) genome-nexus-annotation-pipelines and that build failed, but as part of the build the target directories were cleaned. So that left us with no production jar available. The solution is to rebuild (from an older cbioportal timepoint than I was using) the annotator.jar and to relocate it into /data/portal-cron/lib so that it will not be affected by work which happens in the clone.
